doc/tutorials/validation.md

```diff
@@ -133,41 +133,110 @@ kdb setmeta spec/tutorial/spec/test check/validation/match LINE
 kdb setmeta spec/tutorial/spec/test check/validation/message "Not a number"
 ```
 
-Alternatively, we could mount a plugin that supports metadata,
-e.g. the `ni` plugin, and specify the configuration
-using a text editor (or `cat`):
-
+If we now set a new key with
+```sh
+kdb set /tutorial/spec/test "not a number"
+#> Using name user/tutorial/spec/test
+#> Create a new key user/tutorial/spec/test with string not a number
+```
+this key has adopted all metadata from the spec namespace:
 ```sh
-kdb mount spec.ini spec/tutorial/spec ni
-cat << HERE > `kdb file spec/tutorial/spec`
-[test]
-check/validation = [1-9][0-9]*
-check/validation/match = LINE
-check/validation/message = Not a number
-HERE
 kdb lsmeta /tutorial/spec/test
 #> check/validation
 #> check/validation/match
 #> check/validation/message
 ```
+Note that this key should not have passed the validation that we defined in the
```

Comment:
Avoid "Note that". If it is really a note, you can put it with `> ` to highlight it.